pkg/internal/mock/kms/mock_kms.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package legacykms
```

Comment:
We should expose certain mocks in aries so we can use them.

Reply:
done